src/ssa_tracker.rs

```diff
@@ -46,9 +48,11 @@ impl SSATracker {
                 let skipped_idx = self.current[key];
                 if skipped_idx != *value {
                     let new_ssa = self.allocate_new_ssa_index_by_id(*key);
+                    // TODO This does not work for memory
```

Comment:
Why not?

Reply:
This todo is from before I introduced the types ;)

Reply:
So should it be removed now?

Reply:
@chriseth